index.html

```diff
@@ -66,7 +66,140 @@ <h5>Pay as You Go</h5>
     <li><a href="https://instagram.com"><img src="img/instagram-icon.svg" alt="Instagram Logo"> </a></li>
    </ul>
    <span><h5> &copy; Karma Mobility, Inc.</h5></span>
- </footer>  
- 
+ </footer>
+</body>
+</html>
+
+
+<meta charset="UTF-8">	    <meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">	    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<title>Karma - Store</title>	    <title>Karma - Store</title>
+<link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>	    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+<link rel="stylesheet" href="css/normalize.css">	    <link rel="stylesheet" href="css/normalize.css">
+<link rel="stylesheet" href="css/style.css">	    <link rel="stylesheet" href="css/style.css">
+<link rel="stylesheet" href="/css/store.css">	    <link rel="stylesheet" href="/css/store.css">
+<link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">	    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+</head>	</head>
+<body>
+ <header>
+    <a class="logo" href="index.html"><img src="img/karma-logo.svg" alt="karma logo"></a>
+    <nav>
+        <ul class="list">
+            <li class="items"><a class="links" href="#"> Meet Karma</a></li>
+            <li class="items"><a class="links" href="#"> How it Works</a></li>
+            <li class="items"><a class="links" href="#"> Store</a></li>
+            <li class="items"><a class="links" href="#"> Blog</a></li>
+            <li class="items"><a class="links" href="#"> Help</a></li>
+            <li class="items"><a class="links" href="#"> Login</a></li>  
+        </ul> 
+    </nav>
+ </header> 
+
+ <main class="main2">
+
+    <section class="section2">
+        <h1>Order your Karma wifi device today!</h1>
+
+        <form action="#">
+            <div class="fieldset">
```

Comment:
Call this class `fieldset` is a bit confusing as the div isn't a fieldset. The [`fieldset` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) is used to group several checkboxes or radio buttons together.